### ISSUE TITLE
About the solution of problems like "have 3 channels, but got 1000 channels instead"

### ISSUE THREAD

**Issue by playpopi** [Beckschen/TransUNet#149]
The original code based on 3 channels,and the author copy the input images 3 times to fit the format,so you need to change somewhere to fit your dataset.But I think understanding the changes in data in the model is crucial for using deep learning.
You need to change the input of the ResNet,
vit_seg_modeling_resnet_skip:
      class ResNetV2(nn.Module):
            self.root = nn.Sequential(OrderedDict([
                                    ('conv', StdConv2d("channel num", width, kernel_size=7, stride=2, bias=False, padding=3)),
and ban the pretrained model :train:net.load_from(weights=np.load(config_vit.pretrained_path)),that model is only suitable for fixed parameters.Perhaps there are other issues that need to be addressed by searching for "channel" to make changes.
After all,it's a basic problem,just to document the difficulties encountered during the learning process, my english is poor, glad it may be helpful to you.
